### PR TITLE
RAC-11: Add quantified associations to the InternalApi normalizer

### PR DESCRIPTION
**Pull request by tseho**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Pim/Enrichment/Bundle/EventSubscriber/EntityWithQuantifiedAssociations/LoadEntitySubscriber.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ public function __construct(
     public function getSubscribedEvents()
     {
         return [
-            Events::postLoad => 'loadQuantifiedAssociations'
```
Comment (SamirBoulil) [inline src/Akeneo/Pim/Enrichment/Bundle/EventSubscriber/EntityWithQuantifiedAssociations/LoadEntitySubscriber.php:42]: 😽

Comment (tseho) [reply src/Akeneo/Pim/Enrichment/Bundle/EventSubscriber/EntityWithQuantifiedAssociations/LoadEntitySubscriber.php:42]: :eyes:

File: src/Akeneo/Pim/Enrichment/Component/Product/Connector/ArrayConverter/StandardToFlat/Product.php [hunk 2]
```diff
@@ -35,6 +35,9 @@ protected function convertProperty($property, $data, array $convertedItem, array
             case 'associations':
                 $convertedItem = $this->convertAssociations($data, $convertedItem);
                 break;
+            case 'quantified_associations':
+                // @todo https://akeneo.atlassian.net/browse/RAC-198
```
Comment (StevenVAIDIE) [inline src/Akeneo/Pim/Enrichment/Component/Product/Connector/ArrayConverter/StandardToFlat/Product.php:39]: You reference several times (in CE and EE) this ticket RAC-198 but it does not exist. It's RAC-19 ? or maybe MET-198 ?

Comment (tseho) [reply src/Akeneo/Pim/Enrichment/Component/Product/Connector/ArrayConverter/StandardToFlat/Product.php:39]: s**t, yeah it's MET-198

**Pull request merged**